koku/masu/processor/ocp/ocp_report_parquet_summary_updater.py

```diff
@@ -144,6 +144,7 @@ def update_summary_tables(self, start_date, end_date, **kwargs):
             accessor.populate_pod_label_summary_table([report_period_id], start_date, end_date)
             accessor.populate_volume_label_summary_table([report_period_id], start_date, end_date)
             accessor.update_line_item_daily_summary_with_enabled_tags(start_date, end_date, [report_period_id])
+            accessor.update_line_item_daily_summary_with_tag_mapping(start_date, end_date, [report_period_id])
```

Comment:
Yeah, I think you bring up a good point here. So, what I am wavering on at the moment is that my future hope is to remove the `update_line_item_daily_summary_with_enabled_tags` flow altogether with this tech debt item [COST-4657](https://issues.redhat.com/browse/COST-4657). I think it would be better to filter out disabled tags during the daily summary sql during the Trino to Posgresql transition. Do you think we should make the argument to move that tech debt item into the epic to the team?

Reply:
Yea, it's probably a good idea to pull that in if we can, especially if it will eliminate a known issue with our processing today.